app/screens/claim/index.tsx

```diff
@@ -28,71 +45,89 @@ const Claim = () => {
   const getData = async (result: any) => ({
     hash: await Crypto.digestStringAsync(
       Crypto.CryptoDigestAlgorithm.SHA256,
-      result.uri
+      result.uri,
     ),
-    base64Url: result.uri
+    base64Url: result.uri,
   });
 
   const uploadFileFromBrowser = async () => {
     try {
-      const libraryUrl = await AsyncStorage.getItem("library_url");
-      if(libraryUrl === null) {
-        let result = await DocumentPicker.getDocumentAsync();
-        if (result.type === "success") {
-          await AsyncStorage.setItem("document_url", JSON.stringify(await getData(result)));
-        }
-      } else {
-        // TODO waiting for message
-        alert('You have already selected one document');
+      let result = await DocumentPicker.getDocumentAsync();
```

Comment:
@ahinchliff be good if you could also make the suggestion like this

```suggestion
      const result = await DocumentPicker.getDocumentAsync();
```